src/fundamentals.html

```diff
@@ -409,7 +409,7 @@ <h5 id="fund_units">Length Valued Attributes</h5>
 
     <p>In addition, the attributes on <a href="#presm_mpadded">mpadded</a>
      allow three <dfn>pseudo-units</dfn>, <code>height</code>,
-      <code>depth</code> and <code>width</code> (taking the place of one of the usual CSS units)
+      <code>depth</code>, and <code>width</code> (taking the place of one of the usual CSS units)
```

Comment:
Do we have an official position on the use of the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) in the specification?

Should we?

Reply:
@dginev I don't think we do, or are necessarily consistent. It does appear elsewhere in the spec, eg

>  Identifiers can include variables, function names, and symbolic constants

I don't like it much, despite living in Oxford, but I can live with it either way.